x-pack/auditbeat/module/system/_meta/kibana/6/dashboard/auditbeat-system-host-overview.json

```diff
@@ -0,0 +1,640 @@
+{
+  "objects": [
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "index": "auditbeat-*",
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "Host Count [Auditbeat System]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [
+            {
+              "enabled": true,
+              "id": "1",
+              "params": {
+                "customLabel": "Hosts",
+                "field": "system.audit.host.id.keyword"
+              },
+              "schema": "metric",
+              "type": "cardinality"
+            }
+          ],
+          "params": {
+            "addLegend": false,
+            "addTooltip": true,
+            "metric": {
+              "colorSchema": "Green to Red",
+              "colorsRange": [
+                {
+                  "from": 0,
+                  "to": 10000
+                }
+              ],
+              "invertColors": false,
+              "labels": {
+                "show": true
+              },
+              "metricColorMode": "None",
+              "percentageMode": false,
+              "style": {
+                "bgColor": false,
+                "bgFill": "#000",
+                "fontSize": 60,
+                "labelColor": false,
+                "subText": ""
+              },
+              "useRanges": false
+            },
+            "type": "metric"
+          },
+          "title": "Host Count [Auditbeat System]",
+          "type": "metric"
+        }
+      },
+      "id": "cf8aa0a0-18ca-11e9-9094-c50574723088",
+      "type": "visualization",
+      "updated_at": "2019-01-17T13:02:07.570Z",
+      "version": 8
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "index": "auditbeat-*",
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "OS Distribution [Auditbeat System]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [
+            {
+              "enabled": true,
+              "id": "1",
+              "params": {
+                "field": "system.audit.host.id.keyword"
```

Comment:
Hmmm, the canonical fields are not `keyword` datatype? Is this a fact across Auditbeat?

In ECS, we're flipping the ES convention around. Virtually all textual fields are `keyword`, and if full text search is needed, a multi-field named `.text` should be added. To take this field for example, I would have expected `system.audit.host.id` ==` keyword` and `system.audit.host.id.text` == `text`.

I thought the ECS convention (at least the canonical fields being `keyword`) was already in place across all beats, but it seems like it's not the case here.

@cwurm Do you know how widely multi-field is used for the keyword indexing, in Auditbeat?

cc @ruflin